### ISSUE TITLE
homepage.js line 78 not needed, also creates bug in mobile responsiveness

### ISSUE THREAD

**Issue by codeturtle00** [uoftblueprint/uoftblueprint.github.io#16]
**Line in question:**
`line 78: {window.innerWidth > 480 ? (`

**Results in:**
![image](https://user-images.githubusercontent.com/48864709/100811971-f3871b80-3409-11eb-88cf-42d1cb0a94a1.png)

**Explanation:**
the css for id='values-horizontal' already handles mobile correctly, it was my bad for not naming the id correctly, it should've been named to 'values-grid' or something because it switches between horizontal and vertical using css @media

TLDR removing this line will fix a responsiveness bug.
